book_source/03_topical_pages/94_docker/02_quickstart.Rmd

```diff
@@ -1,4 +1,37 @@
-## The PEcAn docker install process in detail {#docker-quickstart}
+## Quick-start docker install {#docker-quickstart}
+
+```bash
+git clone git@github.com/pecanproject/pecan
+cd pecan
+
+# start database
+docker-compose -p pecan up -d postgres
+
+# add example data (first time only)
+docker-compose run --rm bety initialize
+docker run -ti --rm --network pecan_pecan --volume pecan_pecan:/data --env FQDN=docker pecan/data:develop
+
+# start PEcAn
+docker-compose -p pecan up -d
+
+# run a model
+curl -v -X POST \
+    -F 'hostname=docker' \
+    -F 'modelid=5000000002' \
```

Comment:
```suggestion
    -F 'modelid=14' \ # ED v2.2.0
```

Reply:
@dlebauer I don't see any model with an `id = 14` (e.g., http://welsch.cyverse.org:8000/bety/models/14)

Reply:
only reason why i had `5000000002` is this is sipnet, if you change the model, you need to make sure to change the pft etc as well. Finally sipnet runs quickly so this is a nice get started model.

Reply:
I agree with @robkooper  that this better stays sipnet in the main documentation